src/MICore/Transports/ServerTransport.cs

```diff
@@ -36,15 +36,26 @@ public override void InitStreams(LaunchOptions options, out StreamReader reader,
             proc.StartInfo.FileName = localOptions.DebugServer;
             proc.StartInfo.Arguments = localOptions.DebugServerArgs;
             proc.StartInfo.WorkingDirectory = miDebuggerDir;
-            _startPattern = localOptions.ServerStarted;
+
+            if (!String.IsNullOrWhiteSpace(localOptions.ServerStarted))
+            {
+                _started = false;
+                _startPattern = localOptions.ServerStarted;
+            }
+            else
+            {
+                _started = true;
+            }
+            
             _messagePrefix = Path.GetFileNameWithoutExtension(localOptions.DebugServer);
 
             InitProcess(proc, out reader, out writer);
         }
 
         protected override string FilterLine(string line)
         {
-            if (!_started && Regex.IsMatch(line, _startPattern, RegexOptions.None, new TimeSpan(0, 0, 0, 0, 10) /* 10 ms */))
+            Debug.Assert(_startPattern != null, "Why is _startPattern null");
+            if (!_started && _startPattern != null && Regex.IsMatch(line, _startPattern, RegexOptions.None, new TimeSpan(0, 0, 0, 0, 10) /* 10 ms */))
```

Comment:
I think you want to allow startpattern to be null and treat it as a match when it is, so
(!_started && (_startPattern == null || Regex...))

Reply:
@paulmaybee Should I treat _startPattern the same if it is an Empty string?

Reply:
yes